### PR TITLE
add movemove and wasmwasm

### PR DESCRIPTION
**Pull request by JSHan94**

### CODE REVIEW

**Comment by Vritra4** [comment]
fix urls please

File: devnets/movemove/chain.json [hunk 1]
```diff
@@ -76,19 +76,19 @@
...
       }
     ],
     "rest": [
       {
-        "address": "https://lcd.stonemove-14.initia.xyz",
+        "address": "https://lcd.movemove-1.initia.xyz",
         "provider": "Foundation"
       }
     ],
     "api": [
```
Comment (Vritra4) [inline devnets/movemove/chain.json:89]: remove this

File: devnets/movemove/chain.json [hunk 2]
```diff
@@ -102,15 +102,15 @@
   "explorers": [
```
Comment (Vritra4) [inline devnets/movemove/chain.json:102]: invalid layers here and some below

File: devnets/wasmwasm/chain.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "../../chain.schema.json",
   "chain_name": "Miniwasm",
-  "chain_id": "stonewasm-14",
+  "chain_id": "wasmwasm-14",
```
Comment (Vritra4) [inline devnets/wasmwasm/chain.json:4]: wasmwasm-1 is correct

Comment (Vritra4) [reply devnets/wasmwasm/chain.json:4]: and there are so many wasmwasm-14 lol

File: devnets/wasmwasm/chain.json [hunk 4]
```diff
@@ -97,15 +92,15 @@
   "explorers": [
     {
       "kind": "explorer",
-      "url": "https://explorer.stonewasm-14.initia.xyz/?layer=minimove",
-      "tx_page": "https://explorer.stonewasm-14.initia.xyz/tx/${txHash}?layer=minimove",
-      "account_page": "https://explorer.stonewasm-14.initia.xyz/address/${accountAddress}?layer=minimove"
+      "url": "https://explorer.wasmwasm-14.initia.xyz/?layer=minimove",
```
Comment (Vritra4) [inline devnets/wasmwasm/chain.json:95]: invalid layer again

**Review by ALPAC-4 — APPROVED** [review]

**Pull request merged**